lib/web/apiserver.go

```diff
@@ -3435,6 +3437,8 @@ func (h *Handler) clusterUnifiedResourcesGet(w http.ResponseWriter, request *htt
 
 	getUserGroupLookup := h.getUserGroupLookup(request.Context(), clt)
 
+	clusterComponentFeatures := h.getAllClusterComponentFeatures(request.Context())
```

Comment:
The `getAllClusterComponentFeatures` is still confusing it doesn't return all component features just AuthAndProxy without db agent etc.